src/lib/trading-pairs.js

```diff
@@ -0,0 +1,41 @@
+'use strict'
+const _ = require('lodash')
+
+class TradingPairs {
+  constructor (pairs) {
+    this._pairs = pairs
+  }
+
+  getPairs () {
```

Comment:
Maybe this should be `toArray`. As is, you have things like `pairs.getPairs` which is confusing: "I thought I already had pairs, why would call `getPairs` on `pairs`?"